src/goto-analyzer/get_function_pointer_values.cpp

```diff
@@ -0,0 +1,113 @@
+/*******************************************************************\
+
+Module: Show function pointer values
+
+Author: Diffblue Ltd, 2020
+
+\*******************************************************************/
+
+#include "get_function_pointer_values.h"
+
+#include <util/message.h>
+#include <util/namespace.h>
+
+#include <goto-programs/goto_model.h>
+#include <goto-programs/restrict_function_pointers.h>
+
+#include <analyses/ai.h>
+#include <analyses/variable-sensitivity/abstract_enviroment.h>
+#include <analyses/variable-sensitivity/value_set_abstract_object.h>
+#include <analyses/variable-sensitivity/variable_sensitivity_domain.h>
+
+#include <regex>
+#include <string>
+
+void get_function_pointer_values(
```

Comment:
:bulb: This function would be unit testable (and more generally usable) if you have it return the map, and have the caller write to the file